backend/canisters/cycles_dispenser/impl/src/lifecycle/inspect_message.rs

```diff
@@ -5,12 +5,6 @@ use ic_cdk::inspect_message;
 fn inspect_message() {
     let method_name = ic_cdk::api::call::method_name();
 
-    // 'inspect_message' only applies to ingress messages so calls to c2c methods should be rejected
-    let is_c2c_method = method_name.starts_with("c2c");
```

Comment:
I removed the handling of Frozen so that anyone calling gets back the correct response rather than the call getting rejected.